tests/DocumentationExamplesTest.php

```diff
@@ -728,6 +729,45 @@ public function testExample_42_50(): void
             $this->assertObjectNotHasAttribute('size', $document);
             $this->assertCount(1, $document->instock);
         }
+
+        // Start Aggregation Projection Example 1
```

Comment:
The examples are rendered within [Project Fields with Aggregation Expressions](https://www.mongodb.com/docs/manual/tutorial/project-fields-from-query-results/#project-fields-with-aggregation-expressions), which follows the earlier examples in this method and shares its data fixtures.